### PR TITLE
Conserve line breaks in training and trainer description

### PR DESCRIPTION
**Pull request by Victor-van-Duijnen**
PR introducing the conservation of line breaks in the descriptions fields of Trainings and Trainers in the showcase project.

### CODE REVIEW

File: src/UserAdmin/src/Smart.FA.Catalog.Web/wwwroot/js/lib.js [hunk 1]
```diff
@@ -78,10 +78,10 @@ function disableEnterKeyUpKeyPressOnForm() {
...
-    }
-    return true;
-}
+// function ignoreEnterKeyEventHandler(e) {
+//     if (e.keyIdentifier === 'U+000A' || e.keyIdentifier === 'Enter' || e.keyCode === 13) {
+//         e.preventDefault();
+//         return false;
+//     }
+//     return true;
+// }
```
Comment (CelineMarique) [inline src/UserAdmin/src/Smart.FA.Catalog.Web/wwwroot/js/lib.js:87]: Instead of commenting, you could delete it

Comment (Victor-van-Duijnen) [reply src/UserAdmin/src/Smart.FA.Catalog.Web/wwwroot/js/lib.js]: You are right, I removed it

**Review by CelineMarique — APPROVED** [review]

File: src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Extensions/StringExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+﻿namespace Smart.FA.Catalog.Showcase.Web.Extensions;
+
+public static class StringHelper
+{
+    public static string ConserveLineBreaksInHtml(this string input) => input
```
Comment (maxime-poulain) [inline src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Extensions/StringExtensions.cs:5]: Having an extensions method on IHtmlHelper would be more suited then you could just do 

> @Html.ConserveLineBreaks(xxx)

File: src/UserAdmin/src/Smart.FA.Catalog.Web/wwwroot/js/lib.js [hunk 3]
```diff
@@ -42,7 +40,7 @@ function RegisterScrollToFirstElementInError() {
 
 // Handling of the culture switching.
 document.addEventListener("DOMContentLoaded",
-    function() {
+    function () {
```
Comment (maxime-poulain) [inline src/UserAdmin/src/Smart.FA.Catalog.Web/wwwroot/js/lib.js:43]:
```suggestion
    function() {
```

File: src/UserAdmin/src/Smart.FA.Catalog.Web/wwwroot/js/lib.js [hunk 4]
```diff
@@ -57,7 +55,7 @@ function ChangeCulture(culture) {
 
 function RegisterCultureChange() {
     document.getElementById("culture").addEventListener("change",
-        function(event) {
+        function (event) {
```
Comment (maxime-poulain) [inline src/UserAdmin/src/Smart.FA.Catalog.Web/wwwroot/js/lib.js:58]:
```suggestion
        function(event) {
```

File: src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Extensions/StringExtensions.cs (hunk 2 above)
Comment (Victor-van-Duijnen) [reply src/Showcase/src/Smart.FA.Catalog.Showcase.Web/Extensions/StringExtensions.cs]: Good suggestion, I don't like too much string extensions normally when I can make it work without it but I have been lazy apparently, changed it to IHtmlHelper extension indeed!

**Review by maxime-poulain — APPROVED** [review]

**Pull request merged**